src/commands/registryCredentials/identity/AcrPullVerifyStep.ts

```diff
@@ -0,0 +1,66 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { type AuthorizationManagementClient, type RoleAssignment } from "@azure/arm-authorization";
+import { uiUtils } from "@microsoft/vscode-azext-azureutils";
+import { AzureWizardExecuteStep, GenericParentTreeItem, GenericTreeItem, activityFailContext, activityFailIcon, activitySuccessContext, activitySuccessIcon, createUniversallyUniqueContextValue, nonNullValueAndProp, type ExecuteActivityOutput } from "@microsoft/vscode-azext-utils";
+import { type Progress } from "vscode";
+import { createAuthorizationManagementClient } from "../../../utils/azureClients";
+import { localize } from "../../../utils/localize";
+import { type ManagedIdentityRegistryCredentialsContext } from "./ManagedIdentityRegistryCredentialsContext";
+
+export const acrPullRoleId: string = '7f951dda-4ed3-4680-a7ca-43fe172d538d';
+
+export class AcrPullVerifyStep extends AzureWizardExecuteStep<ManagedIdentityRegistryCredentialsContext> {
```

Comment:
We don't have to now, but I think having a `RoleAssignmentVerificationStep` could be helpful across all extensions.

Reply:
Yeah agreed, that would pair well with your other step